Index: scone-server/server.sh
===================================================================
--- scone-server.orig/server.sh
+++ scone-server/server.sh
@@ -35,7 +35,9 @@ if [ -f "$SERVER_PID" ]; then echo "Erro
 
 echo "All error and log messages will be printed to \"$LOG\"..."
 
+pushd /usr/share/scone-server > /dev/null
 setsid $LISP --noinform --load src/server.lisp > "$LOG" 2>&1 &
+popd > /dev/null
 
 server_pid=$!
 echo $server_pid > "$SERVER_PID"
